cargo/private/cargo_build_script.bzl

```diff
@@ -79,6 +79,30 @@ def _pwd_flags(args):
             res.append(arg)
     return res
 
+def _pwd_flags_isystem(args):
+    """Prefix execroot-relative paths of known arguments with ${pwd}.
+
+    Args:
+        args (list): List of tool arguments.
+
+    Returns:
+        list: The modified argument list.
+    """
+    res = []
+    fix_next_arg = False
+    for arg in args:
+        if fix_next_arg:
+            res.append("${{pwd}}/{}".format(arg))
```

Comment:
We should only do this is the next arg is a relative path - could you throw in a call to https://github.com/bazelbuild/bazel-skylib/blob/2b546aff9e92966ac5276e2b747aaf9f02690371/lib/paths.bzl#L236 in the `if` guarding this?

Reply:
you are absolutely right. I missed that, sorry.